### PR TITLE
[GSOD]Removes content that is now in documentation and adds link to docs

### PR DESCRIPTION
**Pull request by HCloward**
## What/Why/How?
Since my updates to the Redoc documentation have been published, I thought it would be a good idea to start updating the README file. One of the main ways I updated it was by removing the content that is now documented and including a link instead. I also did some other minor cleanup. I plan to do more work on the README in another PR.
## Reference
I have a task on my GitHub project board. https://github.com/HCloward/redoc/projects/1#card-67733561
## Testing
n/a
## Screenshots (optional)
n/a
## Check yourself
n/a
- [ ] Code is linted
- [ ] Tested
- [ ] All new/updated code is covered with tests

### CODE REVIEW

**Review by RomanHotsiy — APPROVED** [review]
Great 👏

**Review by adamaltman — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -20,34 +18,29 @@
...
-- Server Side Rendering ready
-- The widest OpenAPI v2.0 features support (yes, it supports even `discriminator`) <br>
-![](docs/images/discriminator-demo.gif)
-- OpenAPI 3.0 support
-- Basic OpenAPI 3.1 support
+- [Multiple deployment options](https://redoc.ly/docs/redoc/quickstart/intro/)
+- [Server-side rendering (SSR) ready](https://redoc.ly/docs/redoc/quickstart/cli/#redoc-cli-commands)
+- [Simple integration with `create-react-app`](https://redoc.ly/docs/redoc/quickstart/react/)
+  
+  [See an example](https://github.com/APIs-guru/create-react-app-redoc)
```
Comment (skadinna) [inline README.md:25]: Could we specify what this is an example of? It looks like it's related to the previous bullet point about the React app, so maybe just combine them somehow.

File: README.md [hunk 2]
```diff
@@ -20,34 +18,29 @@
...
+- [Command-line interface to bundle your docs into a **zero-dependency** HTML file](https://redoc.ly/docs/redoc/quickstart/cli/)
+- Responsive three-panel design with menu/scrolling synchronization
+- Deep linking support
+- Ability to integrate your API introduction into the side menu
+- High-level grouping in side-menu via [`x-tagGroups`](https://redoc.ly/docs/api-reference-docs/specification-extensions/x-tag-groups/) vendor extension
```
Comment (skadinna) [inline README.md:30]:
```suggestion
- High-level grouping in side-menu via [`x-tagGroups`](https://redoc.ly/docs/api-reference-docs/specification-extensions/x-tag-groups/) specification extension
```

Comment (skadinna) [reply README.md:30]: We used to refer to these as vendor extensions, but now we use a slightly different term. :)

File: README.md [hunk 3]
```diff
@@ -64,153 +57,34 @@ Additionally, all the 1.x releases are hosted on our GitHub Pages-based CDN **(d
...
+[Using the Redoc CLI](https://redoc.ly/docs/redoc/quickstart/cli/).
 
 ## Configuration
 
 ### Security Definition location
-You can inject Security Definitions widget into any place of your specification `description`. Check out details [here](docs/security-definitions-injection.md).
+You can inject the Security Definitions widget into any place in your definition `description`.
+For more information, refer to [Security definitions injection](docs/security-definitions-injection.md).
 
 ### Swagger vendor extensions
```
Comment (skadinna) [inline README.md:86]:
```suggestion
### OpenAPI specification extensions
```

File: README.md [hunk 4]
```diff
@@ -64,153 +57,34 @@ Additionally, all the 1.x releases are hosted on our GitHub Pages-based CDN **(d
...
-ReDoc makes use of the following [vendor extensions](https://swagger.io/specification/#specificationExtensions):
+Redoc uses the following [vendor extensions](https://swagger.io/specification/#specificationExtensions):
```
Comment (skadinna) [inline README.md:87]:
```suggestion
Redoc uses the following [specification extensions](https://swagger.io/specification/#specificationExtensions):
```

**Review by skadinna — APPROVED** [review]
This is great! :clap: I only have a couple of small suggestions.

File: README.md [hunk 5]
```diff
@@ -27,7 +27,7 @@
 - Responsive three-panel design with menu/scrolling synchronization
 - Deep linking support
 - Ability to integrate your API introduction into the side menu
-- High-level grouping in side-menu via [`x-tagGroups`](https://redoc.ly/docs/api-reference-docs/specification-extensions/x-tag-groups/) vendor extension
+- High-level grouping in side-menu via [`x-tagGroups`](https://redoc.ly/docs/api-reference-docs/specification-extensions/x-tag-groups/) specification extension
```
Comment (adamaltman) [inline README.md:30]:
```suggestion
- High-level grouping in side-menu with [`x-tagGroups`](https://redoc.ly/docs/api-reference-docs/specification-extensions/x-tag-groups/) specification extension
```

**Pull request merged**